src/DynamoCoreWpf/Views/Core/AnnotationView.xaml.cs

```diff
@@ -183,23 +183,20 @@ private void GroupTextBlock_SizeChanged(object sender, SizeChangedEventArgs e)
         private void GroupTextBox_OnIsVisibleChanged(object sender, DependencyPropertyChangedEventArgs e)
         {
             var textbox = sender as TextBox;
-            if (textbox != null && textbox.Visibility == Visibility.Visible)
-            {
-                //Record the value here, this is useful when title is poped from stack during undo
-                ViewModel.WorkspaceViewModel.DynamoViewModel.ExecuteCommand(
-                   new DynCmd.UpdateModelValueCommand(
-                       System.Guid.Empty, this.ViewModel.AnnotationModel.GUID, "TextBlockText",
-                       GroupTextBox.Text));
+            if (textbox == null || textbox.Visibility != Visibility.Visible) return;
 
-                ViewModel.WorkspaceViewModel.DynamoViewModel.UndoCommand.RaiseCanExecuteChanged();
-                ViewModel.WorkspaceViewModel.DynamoViewModel.RedoCommand.RaiseCanExecuteChanged();
+            //Record the value here, this is useful when title is poped from stack during undo
```

Comment:
and could you clarify this comment a bit more... what value and where are you recording it?